src/dimwits/unit/test/referenceUnit.test.cpp

```diff
@@ -5,11 +5,17 @@ using namespace dimwits;
 
 SCENARIO("referenceUnit"){
   REQUIRE( Meter() == referenceUnit( dimension::length ) );
+  REQUIRE( meter == referenceUnit( dimension::length ) );
```

Comment:
So, `dimwits::meter` and `dimwits::Meter()` are the same thing now, right? I'm not sure the significance of this, but it makes it easier on my end.

Reply:
being a bit pedantic, `dimwits::meter` is an instance of `dimwits::Meter`, so `dimwits::Meter()` is a distinct but equal instance of the class